### DISCUSSION TITLE
RTÉ

### DISCUSSION THREAD

**Discussion by jon-bit** [iptv-org/community#815] [Q&A]
### Channel ID

RTEOne.ie

### Notes

I would like to see Ms.Browns boys(Yes I am Irish)and I don't see one you can get in the USA(Yes I know you can get it on BBC but I would like to see just a little bit more Irish content too). if we can get a RTÉ not Geo blocked that would be AWESOME! 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://fra10-lw.tvmucho.com/flslive/rte-one-hd-hq/playlist.m3u8?token=4337424-871ad08f-6818-4619-bf0a-619a0ee069f2

**Comment by jon-bit** [comment]
sorry didn't work

**Comment by BellezaEmporium** [comment]
then their token is IP-locked... there's an endpoint in which you get a fresh link all the time, though I can't guarantee it'll work if I make a plugin out of it.

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
https://fra10-lw.tvmucho.com/flslive/rte-one-hd-hq/playlist.m3u8?token=4337424-871ad08f-6818-4619-bf0a-619a0ee069f2 (offline)
<img width="829" alt="image" src="https://user-images.githubusercontent.com/7253922/232235294-f24ee634-e995-4b3d-8b4b-e3f82983afe2.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ie.m3u#L20-L21
